tools/libvirt_test.sh

```diff
@@ -82,6 +82,15 @@ BLUEPRINT_FILE=${TEMPDIR}/blueprint.toml
 COMPOSE_START=${TEMPDIR}/compose-start-${IMAGE_KEY}.json
 COMPOSE_INFO=${TEMPDIR}/compose-info-${IMAGE_KEY}.json
 
+# Create SSH key
+SSH_KEY=/tmp/id_rsa
+ssh-keygen -f "${SSH_KEY}" -N "" -q
+
+# Change cloud-init/user-data ssh key
+key=" -ssh-rsa $(sed "1d;38d;s/\n//g" "${SSH_KEY}"| tr -d '[:space:]')"
```

Comment:
The sed can be simlified and made a little better by `sed "$ d;1d" "${SSH_KEY}"` which will remove first and last lines only, without relying on it being exactly 38 lines.